ui/src/app/views/workspace/component.spec.ts

```diff
@@ -21,8 +29,13 @@ import {
   simulateClick,
   updateAndTick
 } from 'testing/test-helpers';
+import {ServerConfigServiceStub} from '../../../testing/stubs/server-config-service-stub';
+import {ServerConfigService} from '../../services/server-config.service';
 
 class WorkspacePage {
+  @ViewChild(WorkspaceShareComponent)
```

Comment:
I actually don't follow this testing structure at all. This `@ViewChild` annotation can't possibly be doing anything here, can it? `shareModal` is also not referenced anywhere in the test.

Reply:
Since it's the modal is now getting opened via the workspace-nav-bar, I would think the test would go there.

Reply:
removed because modal is now getting opened via the workspace-nav-bar